src/main/resources/public/content/privacy_statement.html

```diff
@@ -1,30 +1,124 @@
+</p>
+
+<h4 id="Dauer">Dauer der Speicherung der personenbezogenen Daten</h4>
+<p>
+    Ihre Daten werden nur so lange gespeichert, wie dies unter Beachtung gesetzlicher Aufbewahrungsfristen zur Aufgabenerfüllung erforderlich ist.
+</p>
+
+<h4 id="Rechte">Ihre Rechte</h4>
+<p>
+    Soweit wir von Ihnen personenbezogene Daten verarbeiten, stehen Ihnen als Betroffener nachfolgende Rechte zu:
+</p>
+
+<ul>
+    <li>Sie haben das Recht auf Auskunft über die zu Ihrer Person gespeicherten Daten (Art. 15 DSGVO).</li>
+    <li>Sollten unrichtige personenbezogene Daten verarbeitet werden, steht Ihnen ein Recht auf Berichtigung zu (Art. 16 DSGVO).</li>
+    <li>Liegen die gesetzlichen Voraussetzungen vor, so können Sie die Löschung oder Einschränkung der Verarbeitung verlangen (Art. 17 und 18 DSGVO).</li>
+    <li>Wenn Sie in die Verarbeitung eingewilligt haben oder ein Vertrag zur Datenverarbeitung besteht und die Datenverarbeitung mithilfe automatisierter Verfahren durchgeführt wird, steht Ihnen gegebenenfalls ein Recht auf Datenübertragbarkeit zu (Art. 20 DSGVO).</li>
+    <li>Falls Sie in die Verarbeitung eingewilligt haben und die Verarbeitung auf dieser Einwilligung beruht, können Sie die Einwilligung jederzeit für die Zukunft widerrufen. Die Rechtmäßigkeit der aufgrund der Einwilligung bis zum Widerruf erfolgten Datenverarbeitung wird durch diesen nicht berührt.</li>
+</ul>
+
+<p>
+    Sie haben das Recht, aus Gründen, die sich aus Ihrer besonderen Situation ergeben, jederzeit gegen die Verarbeitung Ihrer Daten Widerspruch einzulegen, wenn die Verarbeitung ausschließlich auf Grundlage des Art. 6 Abs. 1 Buchst. e oder f DSGVO erfolgt (Art. 21 Abs. 1 Satz 1 DSGVO).
+</p>
+
+<h4 id="Beschwerderecht">Beschwerderecht bei der Aufsichtsbehörde</h4>
+<p>
+    Weiterhin besteht ein Beschwerderecht beim Bayerischen Landesbeauftragten für den Datenschutz. Diesen können Sie unter folgenden Kontaktdaten erreichen:<br />
+    Postanschrift: Postfach 22 12 19, 80502 München<br />
+    Adresse: Wagmüllerstraße 18, 80538 München<br />
+    Telefon: 089 212672-0<br />
+    E-Mail: poststelle(at)datenschutz-bayern.de<br />
+    https://www.datenschutz-bayern.de/<br />
+</p>
+
+<h4 id="Weitere">Weitere Informationen</h4>
+<p>
+    Für nähere Informationen zur Verarbeitung Ihrer Daten und zu Ihren Rechten können Sie uns unter den oben (zu Beginn von A.) genannten Kontaktdaten erreichen.
+</p>
+
+
+<h2 id="Internetauftritt">Informationen zum Internetauftritt</h2>
+
+<h4 id="Technische">Technische Umsetzung</h4>
+<p>
+    Die Webserver von Artemis werden durch die <a href="https://www.rbg.tum.de" target="_blank" rel="external noopener noreferrer">Rechnerbetriebsgruppe</a> der Technischen Universität München betrieben.
+    Die von Ihnen im Rahmen des Besuchs unseres Webauftritts übermittelten personenbezogenen Daten werden daher in unserem Auftrag durch das LRZ verarbeitet:
```

Comment:
An uninformed reader might not understand what LRZ stands for, a link or explaination would be helpful